src/MBX/pair_mbx.cpp

```diff
@@ -0,0 +1,751 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   LAMMPS Development team: developers@lammps.org
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------------
+   Contributing author:
+------------------------------------------------------------------------- */
+
+#include "pair_mbx.h"
+#include "atom.h"
+#include "comm.h"
+#include "error.h"
+#include "fix.h"
+#include "force.h"
+#include "math_const.h"
+#include "memory.h"
+#include "modify.h"
+#include "neigh_list.h"
+#include "neigh_request.h"
+#include "neighbor.h"
+#include "respa.h"
+#include "update.h"
+#include <cmath>
+#include <cstring>
+#include <mpi.h>
+
+#include "domain.h"
+
+#define TTMNRG
+
+// subject for removal
+// Systems::DispersionPME()
+// Systems::ElectrostaticsMPI()
+
+using namespace LAMMPS_NS;
+using namespace MathConst;
+
+/* ---------------------------------------------------------------------- */
+
+PairMBX::PairMBX(LAMMPS *lmp) : Pair(lmp)
+{
+  respa_enable = 1;
+  restartinfo = 0;
+  no_virial_fdotr_compute = 1;
+  one_coeff = 1;
+
```

Comment:
This setting is missing since the default is 1 and there is no `single()` method:
```suggestion
single_enable = 0;

```